bindings/r/inc/RExports.h

```diff
@@ -43,12 +46,27 @@
 #pragma GCC diagnostic ignored "-Wextra"
 #pragma GCC diagnostic ignored "-Wignored-qualifiers"
 #endif
+// disable warning for macos
+#if defined(__APPLE__)
+#pragma GCC diagnostic ignored "-Wnonportable-include-path"
+#
```

Comment:
That needs to be "undone" below, after the relevant `#include`, else nobody including this header will get this warning anymore.

Reply:
I agree, but then should we do this also for all other warnings that are ignored ? See the lines above.

Reply:
Great thanks!